submissions/description/cross_site_scripting_xss/template.md

```diff
@@ -0,0 +1,30 @@
+# Cross-Site Scripting
+
+## Overview of the Vulnerability
+
+Cross-Site Scripting (XSS) is a type of injection attack where malicious JavaScript is injected into a website. When a user visits the affected web page, the Javascript executes within that user’s browser in the context of the domain. XSS can be found on this domain which allows an attacker to control code that is executed within a user’s browser.
```

Comment:
The final sentence for this feels like a runoff, may be worth removing and making it

> **Cross-Site Scripting (XSS) is a type of injection attack where malicious JavaScript is injected into a website. When a user visits the affected web page, the Javascript executes within that user’s browser in the context of the domain.**